gcc/rust/lex/rust-lex.h

```diff
@@ -204,186 +205,6 @@ class Lexer
   Linemap *get_line_map () { return line_map; }
   std::string get_filename () { return std::string (input.get_filename ()); }
 
-  // Input source wrapper thing.
-  class InputSource
```

Comment:
This class has been moved to new file `rust-input-source.h` to avoid recursive `#include`